nilearn/regions/tests/test_region_extractor.py

```diff
@@ -155,6 +155,12 @@ def test_region_extractor_fit_and_transform():
     assert_not_equal(extract_ratio.regions_img_, '')
     assert_true(extract_ratio.regions_img_.shape[-1] >= 9)
 
+    # No region for too high FWHM
```

Comment:
>No region for too high FWHM

I would take this as a NOTE in documentation near ```smoothing_fwhm``` and rather not add a test testing whether it raises an error.

I was trying to solve this "zero-size array" problem in PR #1247 by returning am empty array whenever there are no regions found to be extracted.

So, I am not sure whether we should add a test ?